pcsx2/x86/microVU_Analyze.inl

```diff
@@ -646,15 +646,10 @@ __ri int mVUbranchCheck(mV)
 
 			if (mVUlow.branch == 2 || mVUlow.branch == 10) // Needs linking, we can only guess this if the next is not conditional
 			{
-				if (branchType <= 2 || branchType >= 9) // First branch is not conditional so we know what the link will be so we can let the existing evil block do its thing! We know where to get the addr :)
+				//First branch is not conditional so we know what the link will be
+				//So we can let the existing evil block do its thing! We know where to get the addr :)
```

Comment:
spaces after // please
example
// This is a comment

Reply:
Fixed